src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -278,11 +278,13 @@ const FilterForm = ({
             options={stadsdeel}
           />
 
-          <RadioGroup
+          <CheckboxGroup
             options={priority}
             name="priority"
+            hasToggle={false}
             defaultValue={state.options.priority}
-            onChange={onRadioChange}
+            onChange={onGroupChange}
+            onToggle={onGroupToggle}
```

Comment:
This prop isn't a required prop and can be removed since the group doesn't have a toggle.

Reply:
gedaan

Reply:
maar dat werkt het niet meer